ui/ui-expenses/src/store/drivers/pica.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env browser */
+import pica from 'pica/dist/pica'
+
+const makeReducingCanvas = (img, coeff = 1) => {
```

Comment:
make `coeff` a constant now. 
You can rename it `const BASE_COEFF = 2`

Reply:
if I hard code coeff, it's not necessary to use GLOBAL variable

Reply:
this is not global, this is scoped to the file (pica)

Reply:
yes, global at file. why use this if it's used one time ?